opendcs-rest-api/src/main/java/org/opendcs/odcsapi/dao/OrganizationDao.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright 2025 OpenDCS Consortium and its Contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License")
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.opendcs.odcsapi.dao;
+
+import java.util.List;
+
+public interface OrganizationDao
+{
+
+	List<String> retrieveOrganizationIds() throws DbException;
```

Comment:
We'll be doing the same "nesting" that CWMS Does. This should start with an object that include the name, description, and parent fields.

Also needs to extend the "OpenDcsDao" marker interface.

Reply:
I opted for turning the field off if the reference db is used. A default "organization" seems like it would cause confusion.